src/main/java/com/backbase/ct/bbfuel/setup/AccessControlSetup.java

```diff
@@ -200,8 +200,14 @@ protected void ingestDataGroupArrangementsForServiceAgreement(String internalSer
                 .orElse(null);
 
             if (existingDataGroup == null) {
-                List<ArrangementId> arrangementIds = this.productSummaryConfigurator.ingestArrangements(
-                    externalLegalEntityId, productGroupSeed);
+                List<ArrangementId> arrangementIds;
+                if (isRetail) {
+                    arrangementIds = this.productSummaryConfigurator.ingestRetailArrangements(
+                            externalLegalEntityId);
+                } else {
+                    arrangementIds = this.productSummaryConfigurator.ingestArrangements(
```

Comment:
If it is not retail what kind of arrangements are these? I think it would be better to make random/static data decision on something else. Business arrangements could be loaded statically as well

Reply:
It shouldn't have any difference, to be honest. But, in order not to break other teams who are relying on EUR arrangements and the random product seed.

Reply:
if you keep the product group seed as default way of data generation you will not break anything

Reply:
but i won't have as much control of the data because the way that the code is structured in the productConfigurator. The json file that was present in this PR is just a first iteration. Product is providing more input so the data doesn't gibberish like you saw.

Reply:
i don't see the added value of having the same named accounts for all retail users. If you have a small environment with only a couple of them i would suggest to refactor the product seed group mechanism and make it more flexible on that point. The solution as proposed now cannot be the default.